RELEASE-NOTES.txt

```diff
@@ -1,8 +1,13 @@
 14.3
 -----
-* Block editor: Add support for upload options in Gallery block
 * Aztec and Block Editor: Fix the presentation of ordered lists with large numbers.
 * Added Quick Action buttons on the Site Details page to access the most frequently used parts of a site.
+* Block editor: Add support for changing image sizes in Image blocks
+* Block editor: Add support for upload options in Gallery block
+* Block editor: Added the Button block
+* Block editor: Add scroll support inside block picker and block settings
+* Block editor: Fix issue where adding emojis to the post title added strong HTML elements to the title of the post
+* Block editor: Fix issue where alignment of paragraph blocks was not respected when splitting the paragraph or reading from the original Post content
```

Comment:
:wave: @SergioEstevao ! The "reading from the original Post content" part of this release note is a bit unclear to me. Wondering if we can improve that at all. If any better wording comes to mind feel free to update the release note here yourself. Or if you want to just leave a comment explaining a bit about what that means, I'll see if I can think of any improvements we could make. 🙇